.github/workflows/release.yml

```diff
@@ -122,14 +122,15 @@ jobs:
       - name: Build WASI-NN plugin
         shell: bash
         run: |
-          source /opt/intel/openvino_2021/bin/setupvars.sh
           export Torch_DIR=$(pwd)/${PYTORCH_INSTALL_TO}/libtorch
+          export OpenVINO_DIR=./usr/lib/x86_64-linux-gnu/cmake/openvino2023.0.1
```

Comment:
Ditto.

Reply:
removed this line because it was not need